src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -232,6 +232,7 @@ internal override LexicalSortKey GetLexicalSortKey()
         private sealed class InvokeMethod : SourceDelegateMethodSymbol
         {
             private readonly RefKind refKind;
+            private readonly DelegateDeclarationSyntax syntax;
```

Comment:
> private readonly DelegateDeclarationSyntax syntax; [](start = 11, length = 51)

Member symbols should not hold on to the syntax directly, it should be done through a syntax reference and it looks like base class already does that.